### PR TITLE
user model created, postgre DB configured

### PR DESCRIPTION
**Pull request by yader-code**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -31,3 +31,5 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+.env.*
```
Comment (jrg091) [inline .gitignore:35]:
```suggestion
.env.*

```

File: config/database.yml [hunk 2]
```diff
@@ -5,21 +5,25 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV['DB_HOST'] %>
```
Comment (jrg091) [inline config/database.yml:10]: Please use `Env.fetch` without a default so you will have a clear error on startup

File: db/migrate/20220606192842_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :keywords
+      t.string :name
```
Comment (jrg091) [inline db/migrate/20220606192842_create_users.rb:5]: Can we include some validations so we can test them in the future? I'm thinking on:
* Not null for both cases
* Not empty for both cases
* Keyword is a CSV
* And a complex/weird one so we also have a custom validation, e.g. name should start with a capital letter or another one you would like, as long it has some custom logic I'm fine


Also, can we use some callback here? e.g. to remove trailing and starting white spaces? Or maybe populating a computed columns called `keywrods_count` 😬

**Review by jrg091 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (joaquinco) [reply .gitignore:35]: More context, here's the definition of line: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

**Review by joaquinco — APPROVED** [review]

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  before_save :upcase_first_letters
+  before_save :remove_white_spaces_begin_and_end_of_name
+
+  private
+
+  def upcase_first_letters
+    full_name = name.split
+    self.name = ''
+    full_name.each { |word| self.name += "#{word.capitalize} " }
```
Comment (jrg091) [inline app/models/user.rb:15]: As you have several strings here, `join` will be a usefull function to recreate the name.
```suggestion
    full_name.each { |word| self.name += "#{word.capitalize} " }
```

**Review by joaquinco — APPROVED** [review]

**Review by jrg091 — APPROVED** [review]

**Pull request merged**